language-plutus-core/stdlib/Language/PlutusCore/StdLib/Data/Unit.hs

```diff
@@ -34,3 +35,17 @@ getBuiltinUnitval = rename =<< do
         . TyAbs () a (Type ())
         . LamAbs () x (TyVar () a)
         $ Var () x
+
+-- | @seq @() @()@ as a PLC term.
+--
+-- > \(x y : unit) -> unitval
+getBuiltinSequ :: Quote (Value TyName Name ())
```

Comment:
I don't understand this - we're strict so why do we have this `seq` thing?

Reply:
`emit` on the PLC side has this type: `emit : a -> ()` for some particular `a`. Now if we want to emit two values, we write `sequ (emit x) (emit y)`. So yes, we're strict, but we're not pure and that's why `sequ` exists.